### PR TITLE
Changes to Scope Attribute Management

### PR DESCRIPTION
**Pull request by kavinda1995**
1. Add scope name to attribute update notifications
2. Implement FE based batch updates to attribute updates

### Related Issues
- https://github.com/wso2-enterprise/asgardeo-product/issues/7602

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/oidc-scopes/components/edit-oidc-scope.tsx [hunk 1]
```diff
@@ -140,20 +142,31 @@ export const EditOIDCScope: FunctionComponent<EditScopePropsInterface> = (
...
-    const updateOIDCScope = (attributes: string[]): void => {
+    useEffect(() => {
+        if (!tempSelectedAttributes) {
+            return;
+        }
+
+        setUpdatedClaimList(tempSelectedAttributes);
+    }, [ tempSelectedAttributes ]);
+
+
```
Comment (thivi) [inline apps/console/src/features/oidc-scopes/components/edit-oidc-scope.tsx:153]: Let's remove this line.

**Review by thivi — APPROVED** [review]

File: apps/console/src/features/oidc-scopes/components/edit-oidc-scope.tsx (hunk 1 above)
Comment (kavinda1995) [reply apps/console/src/features/oidc-scopes/components/edit-oidc-scope.tsx]: Done in [65dbbb8](https://github.com/wso2/identity-apps/pull/3384/commits/65dbbb875fdb02e7dd27e83530e73767111b8ee6)

File: apps/console/src/features/oidc-scopes/pages/oidc-scopes-edit.tsx [hunk 2]
```diff
@@ -481,83 +484,86 @@ const OIDCScopesEditPage: FunctionComponent<RouteComponentProps<OIDCScopesEditPa
                                             readOnly={ isReadOnly }
                                         />
                                         { !isReadOnly &&
-                                            <Field.Button
+                                            (<Field.Button
```
Comment (savindi7) [inline apps/console/src/features/oidc-scopes/pages/oidc-scopes-edit.tsx:487]: Shall we add the mandatory form id to the form here? Refer - https://github.com/wso2-enterprise/asgardeo-app-extensions/pull/2021

Comment (kavinda1995) [reply apps/console/src/features/oidc-scopes/pages/oidc-scopes-edit.tsx:487]: Already added by https://github.com/wso2/identity-apps/commit/b92965577dd499eae7eb86273a8fd020fc53e269

**Pull request merged**